examples/full-example/app.js

```diff
@@ -47,4 +47,27 @@ function bundle(req, bundle, variations) {
     return '<script src="'+url+'"></script>';
 }
 
+function entryMap(req, bundle, variations) {
+    // req.mendel.getBundleEntries contains all bundles as keys and array of
+    // entries that were used and normalized by variations, this allows apps
+    // to do their specific logic with their bundles
+    var bundles = req.mendel.getBundleEntries();
+
+    // In this particular case, we used "temporary" on the .mendelrc
+    // go expose our lazy bundle entries from extractify
+    var lazy =  [
+        '<script>',
+        '   (function(){',
+        '       var nameSpace = "_extractedModuleBundleMap";',
+        '       var url = "'+req.mendel.getURL(bundle, variations)+'";',
```

Comment:
So there is no get away of using `findTreeForVariations` since it is used by `req.mendel.getURL`, right. At the end of the day #34 could make 3 `findTreeForVariations` calls as well.